scripts/version.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# This script will test if you have given a leap year or not.
```

Comment:
I don't think this is related to what this is doing.

Reply:
haha copy & 🍝